python/pyspark/mllib/clustering.py

```diff
@@ -690,6 +690,21 @@ class LDAModel(JavaModelWrapper):
     >>> model = LDA.train(rdd, k=2)
     >>> model.vocabSize()
     2
+    >>> topics = model.describeTopics()
+    >>> len(topics)
+    2
+    >>> len(list(topics[0])[0])
+    2
+    >>> len(list(topics[0])[1])
+    2
+    >>> topics = model.describeTopics(1)
```

Comment:
One test case should be enough.

These test will become example in docs, so it's better to be more readable, for example:
```
>>> list(sorted(model.describeTopics()))
[([0, 1], [0.50..., 0.49...]), ([1, 0], [0.50..., 0.49...])]
```